### PR TITLE
postcss-blank : browser compat

### PR DESCRIPTION
**Pull request by romainmenke**

### CODE REVIEW

File: plugins/css-blank-pseudo/src/browser.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
-/* global document,MutationObserver */
+/* global document,window,self,MutationObserver */
 import isValidReplacement from './is-valid-replacement.mjs';
 
 // form control elements selector
-const BLANK_CANDIDATES = 'input,select,textarea';
+function isFormControlElement(element) {
```
Comment (romainmenke) [inline plugins/css-blank-pseudo/src/browser.js:5]: trivial equivalent for `matches()` that isn't a polyfill and thus doesn't require testing or validation

File: plugins/css-blank-pseudo/src/browser.js [hunk 2]
```diff
@@ -108,39 +114,71 @@ export default function cssBlankPseudoInit(opts) {
 
 	const handler = generateHandler(options.replaceWith);
 
-	document.body.addEventListener('change', handler);
-	document.body.addEventListener('input', handler);
-
-	// observe value changes on <input>, <select>, and <textarea>
-	const window = (document.ownerDocument || document).defaultView;
+	if (document.body) {
```
Comment (romainmenke) [inline plugins/css-blank-pseudo/src/browser.js:117]: if the activation is called in the html head `document.body` will still be `undefined`

File: plugins/css-blank-pseudo/src/browser.js [hunk 3]
```diff
@@ -108,39 +114,71 @@ export default function cssBlankPseudoInit(opts) {
...
-						if (node.nodeType === 1 && node.matches(BLANK_CANDIDATES)) {
+						if (node.nodeType === 1 && isFormControlElement(node)) {
 							handler({ target: node });
 						}
 					},
 				);
 			});
 		}).observe(document, { childList: true, subtree: true });
+	} else {
+		window.addEventListener('load', () => {
```
Comment (romainmenke) [inline plugins/css-blank-pseudo/src/browser.js:157]: If the init is called in the html head we miss the initial state.
Adding a few event callbacks to make sure the initial state is correct even if we have no support for mutation observer

File: rollup/presets/browser.javascript.js [hunk 4]
```diff
@@ -43,7 +43,7 @@ export function browserJavascript() {
 					rootDir: path.join(process.cwd(), '..', '..'),
 				}),
 				babel(babelConfig),
-				terser(),
+				// terser(),
```
Comment (Antonio-Laguna) [inline rollup/presets/browser.javascript.js:46]: leftover?

Comment (romainmenke) [reply rollup/presets/browser.javascript.js]: yup :)

**Review by Antonio-Laguna — APPROVED** [review]
LGTM! Thanks for doing this!

**Pull request merged**